spec/features/contribute/undergrad_honors_thesis_spec.rb

```diff
@@ -1,8 +1,9 @@
 require 'rails_helper'
 require 'ffaker'
 include Warden::Test::Helpers
+require 'import_export/deposit_type_importer'
```

Comment:
Does autoload via `rails_helper` miss this file, or can this require be omitted?